app/javascript/packs/webcomponents.js

```diff
@@ -0,0 +1 @@
+import '@webcomponents/webcomponentsjs/webcomponents-loader';
```

Comment:
My one reservation here is related to:

https://open-wc.org/building/polyfills-loader.html#why-this-loader

> The official web component polyfills ships with a loader that should be included in your index.html. The challenge with this loader is that also adds language polyfills like Promise and Symbol on browsers that require it. At best this leads to duplicate code when already including these polyfills elsewhere in your app. At worst it will conflict and crash your app. For instance, this is a known issue with babel polyfills.
>
> This loader seeks to solve this issue, it only loads the required web component polyfills. Additionally, the loader uses dynamic imports so that it integrates properly with your app's dependency graph. This works nicely when using the usage option on babel's polyfill transform.

If that's an issue, we can use their loader instead.